src/providers/BackgroundReplacementProvider/index.tsx

```diff
@@ -46,11 +46,9 @@ const BackgroundReplacementProviderContext = createContext<
   BackgroundReplacementProviderState | undefined
 >(undefined);
 
-const BackgroundReplacementProvider: FC<React.PropsWithChildren<Props>> = ({
-  spec,
-  options,
-  children,
-}) => {
+export const BackgroundReplacementProvider: FC<
+  React.PropsWithChildren<Props>
```

Comment:
You can import out the `PropsWithChildren` and it save writing "React." everytime :)

Reply:
These changes are done by script provided by React for migration. 
What you said works but I just don't think it worths the effort to change this just for styling convention.

Reply:
No worries, I dint knew that it was a script, thanks for clarifying 👍